filereader/texttemplate/texttemplate.go

```diff
@@ -21,9 +22,27 @@ func Parse(filename string, funcs template.FuncMap) (*template.Template, error)
 			}
 			return content, nil
 		},
+		"toJSON": func(v interface{}) (string, error) {
```

Comment:
`toJson` is already available through `Masterminds/sprig` according to https://github.com/Masterminds/sprig/blob/dba49a8d3a46ae8522f79e2c0241842d6ab613ca/defaults.go#L66
Could we use the one from sprig instead?